luigi/contrib/datadog.py

```diff
@@ -99,7 +100,13 @@ def _format_task_params_to_tags(self, task):
         return params
 
     def default_event_tags(self):
-        if not self._config.default_event_tags:
-            return []
+        default_tags = []
 
-        return str.split(self._config.default_event_tags, ',')
+        if self._config.default_event_tags:
+            default_tags = default_tags + str.split(self._config.default_event_tags, ',')
+
+        if self._config.environment:
```

Comment:
This one has a default, do you need to `if` check it?

Reply:
You're absolutely right! :)